doc-content/enterprise-only/openshift/template-deploy-optionalmaven-proc.adoc

```diff
@@ -28,3 +28,10 @@ To configure access to a custom Maven repository, set the following parameters:
 If necessary, set additional parameters.
 
 To complete the deployment, follow the procedure in <<template-deploy-complete-{context}-proc>>.
+
+ifeval::["{context}"=="authoring"]
+[IMPORTANT]
+====
+To export built KJAR files from your projects in {CENTRAL} to the external Maven repository, you must also modify the `pom.xml` file in every project. For instructions about modifying the `pom.xml` file for this purpose, see {URL_DESIGNING_BUSINESS_PROCESSES}#export-project-to-maven[_{DESIGNING_BUSINESS_PROCESSES}_].
```

Comment:
Not sure why we are using "also" here. Is the user modifying anything else apart from the pom? I think this sentence will be clearer without the "also".

Reply:
The "also" refers to the procedure that was just described, setting MAVEN_REPO_URL and the credentials. One has to do that and *also* modify pom.xml.